assets/constants.ts

```diff
@@ -539,6 +539,8 @@ export = {
 
   "ENABLE_NEW_STRUCTURE_PLAYERS": false,
 
+  "ENABLE_NEW_STRUCTURE_VIEWER_UPDATES": false,
```

Comment:
Perhaps add a comment explaining precisely what this controls, so that other devs don't use it beyond its intended purpose.

Reply:
Done